binaries/Readme.md

```diff
@@ -111,24 +111,90 @@ Binaries (query engine binary and migration engine binary) are at the core of Ph
+Prisma provides multiple binaries for various platforms and operating systems.
+
+### Why?
+
+We need different binaries because operating systems such as Windows and Linux work 
+fundamentally different. Some Linux distributions have the same issue. On top of that,
+different versions of Linux distributions have different versions of OpenSSL installed,
+which is why we need different binaries for those as well.
+
+### How?
+
+We build multiple binaries on common operating systems with different combinations
+of OpenSSL versions. This results in a few binaries which work on a large selection
+of operating systems, distributions and cloud platforms.
+
+Initially, we built specific binaries for given platforms (e.g. Netlify), which resulted in many 
+binaries which couldn't be shared with similar platforms and it was hard to maintain. 
+
+### Binary builds
+
+The following table lists the pre-built binaries provided by Prisma.
+These include Windows and Mac, and multiple variations for Linux distributions
+
+| Build          | OS                  | OpenSSL variations | Downloads |
+| :------------- | :------------------ | :----------------: | :-------- |
+| `darwin`       | Mac                 | n/a                | (tbd)     |
+| `windows`      | Windows             | n/a                | (tbd)     |
+| `debian`       | Debian              | 1.0.x, 1.1.y       | (tbd)     |
+| `rhel`         | CentOS              | 1.0.x, 1.1.y       | (tbd)     |
+
+### Linux distributions and versions
+
+This table shows which Linux distributions are compatible with our pre-built binaries.
+
+| OS                  | Version                  | Build          | Status [?](#status-legend) | OpenSSL [[1]](#openssl) | Comment |
+| :------------------ | :----------------------- | :------------- | :------------------------: | :---------------------: | :------ |
+| Debian              | 8 (Jessie)               | `debian`       | (:heavy_check_mark:)       | 1.0.x                   | [[2]](#dependencies) install `nodejs` or `libssl1.0.0`
+|                     | 9 (Stretch)              |                | (:heavy_check_mark:)       | 1.1.x                   | [[2]](#dependencies) install `nodejs` or `libssl1.1`
+|                     | 10 (Buster)              |                | (:heavy_check_mark:)       | 1.1.x                   | [[2]](#dependencies) install `nodejs` or `libssl1.1`
+| Ubuntu              | 14.04 (trusty)           | `debian`       | (:heavy_check_mark:)       | 1.0.x                   | [[2]](#dependencies) install `nodejs` or `openssl`
+|                     | 16.04 (xenial)           |                | (:heavy_check_mark:)       | 1.0.x                   | [[2]](#dependencies) install `nodejs` or `openssl`
```

Comment:
In my understanding, Ubuntu 16 + latest libssl version will use the binary from Ubuntu 18. Please test if that works.

Reply:
TLDR: Yes, it works.

Long version: It depends on various factors. Do you mean Ubuntu 16 with the latest OpenSSL version shipped by the package manager for that version (1.0) or the latest OpenSSL version in general (1.1)? Either way, both scenarios _should_ work. Also, we may build OpenSSL variations on the same machine (e.g. 16.04) or we could use different Ubuntu versions because they ship with different defaults, but again it should work fine either way as prisma/binary-tester reported.